packages/main/src/Calendar.hbs

```diff
@@ -2,12 +2,15 @@
 	class="ui5-cal-root"
 	@keydown={{_onkeydown}}
 >
+	<slot name="specialDates"></slot>
```

Comment:
we don't need that slot tag here since special dates are not rendered directly in the calendar.